### PR TITLE
xkore 2 fix male character in iRO

### PR DESCRIPTION
**Pull request by alisonrag**
Fix for male char and unknow map in iRO/bRO

Character Server:
xkore 2 iro char server

Map Server:
xkore 2 iro map server

### CODE REVIEW

**Comment by alisonrag** [comment]
@allanon 
to openkore work with xkore 2 in iro we need to add this in iRO.pm
```
my %handlers = qw(
		received_characters 099D
		received_characters_info 082D
		sync_received_characters 09A0
	);

	$self->{packet_lut}{$_} = $handlers{$_} for keys %handlers;
```
because of this:
src\base\ragnarok\CharServer.pm line 97
`if ($self->{recvPacketParser}{packet_lut}{received_characters} eq '099D') {`

**Comment by lututui** [comment]
Also fixes "unknown map" in bRO

File: src/Base/Ragnarok/CharServer.pm [hunk 1]
```diff
@@ -230,8 +230,14 @@ sub sendCharInfo {
 		my $index = -1;
 		foreach my $char ($self->getCharacters($session)) {
 			$index++;
+			
+			my $sex = "";
+			my $map = $field->baseName.".gat";
```
Comment (lututui) [inline src/Base/Ragnarok/CharServer.pm]: Why use a new var when using it only once?

Comment (lututui) [reply src/Base/Ragnarok/CharServer.pm]: = "" is redundant

**Pull request merged**